client/stylesheets/_modal.scss

```diff
@@ -305,7 +305,7 @@
 .offerRequestMapsApi > div {
 	margin-top: 15px !important;
 	margin-left: 12px !important;
-	height: 280px !important;
+	height: 280px !important; 
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Line contains trailing whitespace